recipes/zyre/2.0.0/conanfile.py

```diff
@@ -0,0 +1,62 @@
+from conans import ConanFile, CMake, tools
+from packaging import version
+import os
+
+
+class ZyreConan(ConanFile):
+    name = "zyre"
+    version = "2.0.0"
+    license = "MPL-2.0"
+    url = "https://github.com/conan-io/conan-center-index"
+    homepage = "https://github.com/zeromq/zyre"
+    description = "Local Area Clustering for Peer-to-Peer Applications."
+    topics = ("conan", "zyre", "czmq", "zmq", "zeromq",
+              "message-queue", "asynchronous")
+    exports_sources = ['CMakeLists.txt',  'patches/*']
+    settings = "os", "compiler", "build_type", "arch"
+    requires = "zeromq/4.3.2", "czmq/4.2.0"
+    options = {
+        "fPIC": [True, False],
+        "drafts": [True, False],
```

Comment:
The zyre project offers the cmake option `ZYRE_BUILD_STATIC`, which means you can add the option `shared` in your recipe. Of course, you will need to remove shared/static library on package step.

Reply:
I also agree with that but unfortunately, this option is just supported with the latest 2.0.1 beta of the master branch and not with the official 2.0.0 release.
I just wanted to create a all package which deals with newer releases later on.

Reply:
Understood, thanks for clarifying it.

Reply:
When zyre 2.0.1 arrives, we can always patch 2.0.0 so the recipes can be merged.